### PR TITLE
Deploy gcsweb on app.ci

### PR DESCRIPTION
**Pull request by alvaroaleman**
/assign @stevekuznetsov 

You can look at it at e.G. https://gcsweb-ci.apps.ci.m4r3.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/8030/pull-ci-openshift-release-master-config/19860/

### CODE REVIEW

File: clusters/app.ci/prow/03_deployment/gcsweb.yaml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      to:
+        kind: ImageStreamTag
+        name: "${NAME}:binaries"
+    source:
+      git:
+        ref: "${SOURCE_REF}"
+        uri: "${SOURCE_URL}"
+      contextDir: "${SOURCE_CONTEXT}"
+      type: Git
+      dockerfile: |
```
Comment (stevekuznetsov) [inline clusters/app.ci/prow/03_deployment/gcsweb.yaml]: Would it be hard to consume from upstream instead ot propagating this?

Comment (alvaroaleman) [reply clusters/app.ci/prow/03_deployment/gcsweb.yaml]: Upstream neither has automated image publishing nor a manifest for gcsweb in their repo.

Comment (stevekuznetsov) [reply clusters/app.ci/prow/03_deployment/gcsweb.yaml]: Using the `prow_image` stuff in Bazel is pretty straightforward -- could we do that, please?

Comment (alvaroaleman) [reply clusters/app.ci/prow/03_deployment/gcsweb.yaml]: yeah, but porting the Makefile is not. I've opened https://github.com/kubernetes/test-infra/issues/17043 for this.

**Pull request merged**